lib/smart_proxy_dynflow_core/api.rb

```diff
@@ -45,9 +51,14 @@ class Api < ::Sinatra::Base
       tasks_count(params['state']).to_json
     end
 
+    post "/tasks/:task_id/update" do |task_id|
```

Comment:
How about matching the route with a regexp and providing a single body since the behavior of `/update` and `/done` is the same?

Reply:
Updated